docs/cadence/language/built-in-functions.mdx

```diff
@@ -29,39 +29,43 @@ Use this function for internal sanity checks.
 
 The message argument is optional.
 
-## `unsafeRandom`
+## `revertibleRandom`
 
 ```cadence
-fun unsafeRandom(): UInt64
+fun revertibleRandom(): UInt64
 ```
 
+<Callout type="info">
+`unsafeRandom()` function currently behaves the same way as `revertibleRandom()`.
+`unsafeRandom` will be deprecated in the next release of Cadence. 
+</Callout>
+
```

Comment:
Looking at the existing docs (e.g. https://developers.flow.com/tools/clients/fcl-js/api#verifyusersignatures-deprecated), I think the convention is that all functions have their onw entry, even if they are deprecated. Why are we not deprecating the function right away? 

```suggestion
## `unsafeRandom` 

This function is superseded by `revertibleRandom()`. For downwards compatibility, `unsafeRandom` still exists but it now backed by the same implementation as `revertibleRandom()`, which has the same API but is significantly more secure. 

<Callout type="info">
`unsafeRandom` will be deprecated in the next release of Cadence
</Callout>
```

Reply:
> Why are we not deprecating the function right away?

It will be deprecated in stable Cadence to avoid the immediate breaking change.

Reply:
After reading [Bastian's comment below](https://github.com/onflow/docs/pull/423#pullrequestreview-1716179181) I think we might need two versions of the document (?):
1. a version for the current cadence, including the `unsafeRandom` function with deprecation notice
2. a version for stable cadence without `unsafeRandom`